packages/frontend/src/pages/contribute.css

```diff
@@ -10,14 +15,13 @@
 .contribute-left {
   flex: 1;
   flex-wrap: wrap;
-
   display: flex;
   flex-direction: column;
-  gap: 16px;
+  gap: 1rem;
 }
 
 .contribute-main p {
-  max-width: 550px;
+  width: 550px;
   line-height: 28px;
 }
```

Comment:
there seems to be some extra padding at the end of the div(s) in the left container.. i have not been able to find where it comes from. I think there should be less space between the text in the left div and the portal image in the right. Also would like the portal image to be a bit bigger for balance but i'm not having luck with that. maybe we need to ask Chiali to create a larger image?